examples/angular/src/app/pages/wallet-selector/wallet-selector.component.ts

```diff
@@ -55,7 +55,7 @@ export class WalletSelectorComponent implements OnInit {
       network: "testnet",
       debug: true,
       modules: [
-        setupMyNearWallet(),
+        setupMyNearWallet({ deprecated: true }),
```

Comment:
Is this deprecation intended?

Reply:
Will remove it it was just for testing, thanks for spotting this.